subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/ShipkitBintrayPluginTest.groovy

```diff
@@ -86,7 +85,9 @@ class ShipkitBintrayPluginTest extends PluginSpecification {
     }
 
     def "prints informative message before upload"() {
-        BintrayUploadTask u = project.tasks[ShipkitBintrayPlugin.BINTRAY_UPLOAD_TASK]
+        //BintrayUploadTask cannot be referenced directly due to runtime error related to changes between Gradle 2 and 3:
```

Comment:
No, it's a strange issue related to the fact that Bintray plugin is built with Gradle 2.4 and it fails in integration tests run with newer Gradle versions by TestKit (it can, as there were incompatible changes in Gradle). I wanted to reproduce it in a separate Gradle project with just the Bintray plugin applied (and newer Gradle version) to report the issue to them, but it worked fine in that configuration.

If you have some space time, you can try to trace the issue and help Bintray team solve it (nevertheless, it hasn't been reported before, therefore, it should be rare).